website/content/community-tools.mdx

```diff
@@ -19,7 +19,7 @@ vendors. These plugins are not officially tested nor officially maintained by
 HashiCorp, and are listed here in order to help users find them easily.
 
 To learn more about how to use community plugins, or how to build your own,
-check out the docs on [extending Packer](/docs/plugins/install-plugins)
+check out the docs on [extending Packer](/packer/docs/plugins/install-plugins)
```

Comment:
To update after https://github.com/hashicorp/dev-portal/pull/1578 is merged

```suggestion
check out the docs on [extending Packer](https://developer.hashicorp.com/packer/docs/plugins/install-plugins)
```

Reply:
As discussed in slack, we will fix in a follow up PR!